src/main/java/net/sf/jabref/logic/formatter/casechanger/ProtectTermsFormatter.java

```diff
@@ -46,11 +56,10 @@ private String format(String text, List<String> listOfWords) {
 
     @Override
     public String format(String text) {
-        Objects.requireNonNull(text);
-
-        if (text.isEmpty()) {
+        if (Objects.requireNonNull(text).isEmpty()) {
             return text;
         }
+        Objects.requireNonNull(ProtectTermsFormatter.protectedTermsLoader);
```

Comment:
Better put this requireNonNull statement in the constructor. The code here fails anyway with a NPE if the terms loader is null.